components/add_new_cv_form.tsx

```diff
@@ -333,11 +401,60 @@ export default function AddNewCvForm({ id }: Props) {
               </h2>
 
               <div className="-my-8 divide-y-2 divide-gray-100 dark:divide-gray-700">
-                <CvFieldArray
-                  fProps={formProps}
-                  title={"Certifications"}
-                  fieldArrayName={"certifications"}
-                />
+                <div className="flex flex-wrap py-8 md:flex-nowrap">
+                  <div className="mb-6 flex flex-shrink-0 flex-col md:mb-0 md:w-64">
+                    <span className="title-font font-semibold text-gray-700 dark:text-gray-400">
+                      Certifications
+                    </span>
+                  </div>
```

Comment:
This looks like too much DOM elements for one label. Let's address this one in follow up PR.